optuna_dashboard/ts/components/StudyDetail.tsx

```diff
@@ -599,12 +599,44 @@ export const TrialTable: FC<{ studyDetail: StudyDetail | null }> = ({
       }
     },
   })
-  columns.push({
-    field: "params",
-    label: "Params",
-    toCellValue: (i) =>
-      trials[i].params.map((p) => p.name + ": " + p.value).join(", "),
-  })
+  if (
+    studyDetail?.union_search_space.length ===
+    studyDetail?.intersection_search_space.length
+  ) {
+    studyDetail?.intersection_search_space.forEach((s) => {
+      columns.push({
+        field: "params",
+        label: `Param ${s.name}`,
+        toCellValue: (i) =>
+          trials[i].params.find((p) => p.name === s.name)?.value || null,
+        sortable: true,
+        filterable: false,
```

Comment:
Could you add `// TODO(yoshinobc): Support filtering by categorical parameters`?